README.md

```diff
@@ -31,9 +35,11 @@ The included symbols libraries so far are:
 * Google Material Design **Places** Set
 * Google Material Design **Social** Set
 * Google Material Design **Toggle** Set
```

Comment:
oops, forgot one thing - it's not super important, so commenting just for the records. do it only if u want to.

please add 2 spaces after the last character of this line (i.e. "Set"). what this will do is add a hard line break after this.

Reply:
Fixed, now it is a table. Additional sets and styles still have problems, i.e. black squares. Any ideas?

Reply:
> Fixed, now it is a table

hmm, that should not be the case. which editor were u using?? it did a whole lots of other things rather than adding 2 spaces 😅. oh wait, did u intentionally did that??

https://github.com/PanderMusubi/inkscape-open-symbols/commit/0d3aa429c02ef8dd408dc6e2aab5d213720751a8#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5



> Additional sets and styles still have problems, i.e. black squares. Any ideas?

where? i cant find that. screenshots?

Reply:
Yes, that was intentionally as Material Design has many more sets. A Home and Search has been added and four additional styles for all sets.

All is in the repo at the moment. You can receate all with the script `1-update-material-design.sh` (only run once as it takes a long time) and `2-update-material-design.sh` (which runs quickly). Changing the style has a lot of influence. Some styles render some icons good for certain settings, others for other style settings. Also a bounding box is shown, which we don't want.

Better to continue the discussion in https://github.com/PanderMusubi/inkscape-open-symbols/issues/117

Reply:
> Better to continue the discussion in #117

oh that yeah, i read that, but couldnt understand. ohkay, will continue there.